### ISSUE TITLE
Track theme popularity

### ISSUE THREAD

**Issue by paracycle** [paracycle/slackthemes#50]
The simplest way to track theme popularity would be to generate synthetic pageview events every time a user-initiated theme change is performed.

**Comment by chteuchteu** [comment]
This would not be great performance-wise (I just applied all the themes one by one to test them).
We could track `#sidebar_theme_custom` input (the ones with the colors codes to copy-paste into Slack) focus event, or even better, detect `Ctrl+C` event when this input is focused.

What do you think?

**Comment by craigotis** [comment]
I think a button to the right of the copy-paste area, titled **"I'm using this!"** would let folks opt-in to the tracking, alleviate any significant performance hit, and make the counts fairly trackable. (Multiple copy-pastes wouldn't count, but multiple clicks would, unless you do unique-by-IP filters, etc.)

**Comment by paracycle** [comment]
Yeah both of these suggestions are great, but I am trying to limit the set of options to ensure that the site stays "serverless", i.e. doesn't need a backend. So, in either case, I will need to find a way to send analytics data to a datastore like Google Analytics, or something like it.

**Comment by FrontEndCoffee** [comment]
It's been a while since the last update in this issue, but I could give this a shot. 

I would suggest we use the [Event Tracking](https://developers.google.com/analytics/devguides/collection/analyticsjs/events) feature in Google Analytics. If you want I can draw up a PR
